src/pkg/packaging/dir.proj

```diff
@@ -10,6 +10,7 @@
       GenerateNugetPackages;
       GenerateInstallers;
       GenerateProjectInstallers;
+      GenerateVSNugetPackages;
```

Comment:
Ah, this also needs to be after `BuildCombinedInstallers` to pick up the MSIs generated by the new tooling like the netstandard targeting pack.

Reply:
Thanks for spotting this - fixing.